docs/development/releasing.rst

```diff
@@ -64,36 +63,19 @@ packages that use the full bugfix/maintenance branch approach.)
    step is only required on major releases, but can be done for bugfix releases
    as time allows.)
 
-#. (Optional) You may want to set up a clean environment to build the release.
-   For more on setting up virtual environments, see :ref:`virtual_envs`, but
-   for the sake of example we will assume you're using `Anaconda`_. This is not
-   necessary if you know your normal python environment has what you need, but
-   you might want to do something like this for safety's sake::
+#. To build the source distribution in an isolated environment and make sure you
+   have all the dependencies required for it, install the `pep517
```

Comment:
You don't actually need all the deps right? `pep517` will install them for you?

Reply:
Indeed - this sentence doesn't say you need to install them yourself though currently? I was trying to imply that pep517 took care of that. Should I rephrase?

Reply:
I think it is okay to keep as-is.